COMETwebapp/ViewModels/Components/Shared/ParameterEditors/ParameterTypeEditorSelectorViewModel.cs

```diff
@@ -50,18 +55,29 @@ public class ParameterTypeEditorSelectorViewModel : IParameterTypeEditorSelector
         /// <param name="parameterType">the <see cref="ParameterType" /> used for this view model</param>
         /// <param name="valueSet">the value set asociated to the ParameterTypeEditor</param>
         /// <param name="isReadOnly">Value asserting that the <see cref="IParameterEditorBaseViewModel{T}" /> should be readonly</param>
-        public ParameterTypeEditorSelectorViewModel(ParameterType parameterType, IValueSet valueSet, bool isReadOnly)
+        public ParameterTypeEditorSelectorViewModel(ParameterType parameterType, IValueSet valueSet, bool isReadOnly, int compoundIndex = -1, EventCallback<IValueSet>? onParameterValueSetChanged = null)

```

Comment:
I would not add the EventCallback inside the constructor, please set it from the CompoundParameterTypeEditorVM when creating it